source/_integrations/emoncms.markdown

```diff
@@ -19,177 +19,17 @@ related:
 
 The `emoncms` sensor {% term integration %} creates sensors for the feeds available in your local or cloud based version of [Emoncms](https://emoncms.org).
```

Comment:
in the header, please add 
ha_config_flow: true

Reply:
done